sites/uat/features/high-value/ds.feature

```diff
@@ -48,7 +48,7 @@ Feature: Display Suite
     Then I should see "Test DS View Mode"
     And I should see "tags"
     And I should see "foo"
-    When I go to "test-ds-view-mode?v=ifbpcnojcqmnmnro"
+    When I go to "test-ds-view-mode?v=ifbpcnojcqmnmnro&foo=bar"
```

Comment:
This snuck in here. It doesn't actually do anything, as this is an authenticated user request.